db/migrate/20200710000238_add_donation_to_inovices.rb

```diff
@@ -0,0 +1,5 @@
+class AddDonationToInovices < ActiveRecord::Migration[5.2]
+  def change
+    add_column :invoices, :donation, :integer
```

Comment:
I'd make this a foreign key to the `donation_id`, which if null, means there was no donation. I somewhat don't like giving this a double meaning so you could even have a flag `has_donation` on invoices to indicate that and if that's true it better have a `donation_id` too.